### PR TITLE
Add pack command to cli

### PR DESCRIPTION
**Pull request by stwiname**
Adds command `subql pack` which webpacks the project code into a single file

Fixes https://github.com/subquery/subql/issues/464

### CODE REVIEW

**Comment by stwiname** [comment]
This somewhat conflicts with the build command and the template project https://github.com/subquery/subql-starter/blob/main/package.json#L8 scripts. Although `rm -rf dist && npm run build` will be run twice, once in the build command and when `npm pack` is called.

I think updating the build command to run webpack would also make sense here

File: packages/cli/src/commands/pack.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        }
+
+        this.log('Finished building code');
+        resolve(true);
+      });
+    });
+
+    // Simplest way to bundle up the files we need into an archive
+    this.log('Packinging your SubQuery Project ...');
+    child.execSync('npm pack');
```
Comment (stwiname) [inline packages/cli/src/commands/pack.ts:94]: I think this makes sense until https://github.com/subquery/subql/issues/463 is complete then we can reassess once we don't need package.json

Comment (ianhe8x) [reply packages/cli/src/commands/pack.ts:94]: We can safely remove `npm pack`, which is not used anyway. the hosted service pull source code directly.

Comment (ianhe8x) [reply packages/cli/src/commands/pack.ts:94]: 👍

**Comment by ianhe8x** [comment]
> This somewhat conflicts with the build command and the template project https://github.com/subquery/subql-starter/blob/main/package.json#L8 scripts. Although `rm -rf dist && npm run build` will be run twice, once in the build command and when `npm pack` is called.
> 
> I think updating the build command to run webpack would also make sense here

They are conflict because we actually created another build approach and use that in pack command. And before we finish everything they will be both available.

**Comment by ianhe8x** [comment]
image

**Comment by ianhe8x** [comment]
Rename bundle to build (replace the current build)
Run `npm pack` or just `oclif-dev manifest && oclif-dev readme` in packages/cli and commit the generated README

**Review by ianhe8x — APPROVED** [review]

**Pull request merged**